apps/web-mzima-client/src/app/shared/components/main/sidebar/sidebar.component.scss

```diff
@@ -15,6 +15,25 @@
   -webkit-overflow-scrolling: touch;
   padding: 24px var(--side-offset) 20px;
 
+  .visibly-hidden {
```

Comment:
Could you move this to styles.scss instead? I think its useful in more places than just here :)